packages/core/core-flows/src/cart/workflows/add-to-cart.ts

```diff
@@ -93,6 +94,24 @@ export const addToCartWorkflow = createWorkflow(
       return (data.input.items ?? []).map((i) => i.variant_id).filter(Boolean)
     })
 
+    const setPricingContext = createHook("setPricingContext", {
+      cart,
+      variantIds,
+      items: input.items,
+      additional_data: input.additional_data,
+    })
+
+    const setPricingContextResult = setPricingContext.getResult() as any
+    const pricingContext = transform(
+      { cart, setPricingContextResult },
+      (data) => {
+        return {
+          ...data.cart,
```

Comment:
I know this was like that before, but maybe we shouldn't pass the entire cart to the pricing module.
We can just use the same fields we are passing on other workflows. 
Apparently there is a util `filterObjectByKeys(data.cart, cartFieldsForPricingContext)` used on other places.

Reply:
So essentially all these workflows should send the same set of properties? Which are

- currency_code
- region_id
- customer_id

Do you think there could be an unintentional breaking change after this?

Reply:
There is a bit more that is passed apparently, including shipping address. basically everything that is declared on that variable `cartFieldsForPricingContext`.

Yes, this is unintentional breaking change, but I don't expect that to be widely used. 
@oli can decide here what is the best to be done. I believe the entire cart can be a huge parameter to be passing all the time, but if we want to avoid even this minimal breaking change, ideally all the other places also receive the full cart.

Reply:
@thetutlage @carlos-r-l-rodrigues, should we perhaps leave a TODO that we can revisit when we can be more strict about what we pass in the pricing context? I'd like to avoid having to decide on this right now AND avoid the breaking change.

Reply:
Yeah, I tried cherry picking the fields and tests started to fail. Ofcourse, I would have missed some, but I can imagine it being an exercise to find which precise data (per workflow) has to be passed down

Reply:
Yeah my thinking too